### ISSUE TITLE
Add CJK subsets fonts support.

### ISSUE THREAD

**Issue by devhaozi** [WordPress/wporg-parent-2021#149]
**Describe**
Hello, I'm the LM of Simplified Chinese (China). I found that the default fonts (EB Garamond and Latin) used in the new theme do not support CJK subsets, so I opened this issue (This isn't a big problem because browsers will automatically handle fallbacks).

For simplified Chinese, I suggest use [Source Han Sans (Noto Sans CJK)](https://github.com/adobe-fonts/source-han-sans), which is already used by Japanese websites and also supports Traditional Chinese and Korean.

Simplified Chinese (Singapore), Traditional Chinese (Hong Kong, Taiwan) and Korean rosetta site also need to be modified, but I am not sure about their LM's decision.

**Comment by ryelle** [comment]
We've already added a CJK serif font for [Japanese](https://ja.wordpress.org/), Noto Serif — for use in place of EB Garamond (headings). Would that work for your site too?

Noto Sans CJK could be added in place of Inter (body font), or you could continue to use the system `sans-serif` fallback — that seems to be what the Japanese site did, perhaps to save bandwidth from loading two CJK font files.

**Comment by devhaozi** [comment]
> We've already added a CJK serif font for [Japanese](https://ja.wordpress.org/), Noto Serif — for use in place of EB Garamond (headings). Would that work for your site too?

Yes, I recommend using Noto Serif instead of EB Garamond for headings (Chinese website also encountered the same problem as the Japanese website [before](https://github.com/WordPress/wporg-main-2022/issues/266#issuecomment-1790935017)).

> Noto Sans CJK could be added in place of Inter (body font), or you could continue to use the system `sans-serif` fallback — that seems to be what the Japanese site did, perhaps to save bandwidth from loading two CJK font files.

Use the system `sans-serif` fallback in body looks ok.

**Comment by Jiwoon-Kim** [comment]
Subject: Switch to Google Variable Fonts for Better Performance on Rosetta Site

Message:

Hi everyone,

I’ve been considering the font options for the Rosetta site, and I believe we can improve both performance and font flexibility by switching to Google Variable Fonts.

Why Google Variable Fonts?

Google Fonts are lightweight and optimized for web use, offering better performance than traditional font loading methods.

They support a range of weights and styles with a single font file, which reduces the number of requests and overall page load times.

The Noto Sans KR font, for example, is available as a variable font, offering excellent support for Korean characters. It would be a great fit for the Rosetta site.


How to Implement:

You can use the following to import Noto Sans KR as a variable font from Google Fonts:

<link href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap" rel="stylesheet">

This approach will load only one font file that covers all weights (from 100 to 900), reducing resource usage and improving load time.

By switching to a variable font like Noto Sans KR, we’ll have a much more efficient font solution for multilingual content on the Rosetta site.

Let me know your thoughts, and if you'd like to discuss implementation further!
